client/packages/sprotty-client/src/features/tool-feedback/change-bounds-tool-feedback.ts

```diff
@@ -148,6 +151,132 @@ export class FeedbackMoveMouseListener extends MouseListener {
         return collisionChain;
     }
 
+    /**
+    * Returns bounds centered around the point
+    */
+    getCenteredBoundsToPointer(mousePoint: Point, bounds: Bounds): Bounds {
+        const middleX = mousePoint.x - bounds.width / 2;
+        const middleY = mousePoint.y - bounds.height / 2;
+        const shiftedBounds: Bounds = { x: middleX, y: middleY, width: bounds.width, height: bounds.height };
+        return shiftedBounds;
+    }
+
+
+    // Remove this and use the one from the improved routing branch
+    getDistanceBetweenParallelLines(p1: Point, p2: Point, secondLine: PointToPointLine): Number {
+        const numerator: number = Math.abs((secondLine.a * p1.x) + (secondLine.b * p1.y) - secondLine.c);
+        const denominator: number = Math.sqrt(Math.pow(secondLine.a, 2) + Math.pow(secondLine.b, 2));
+        return numerator / denominator;
+    }
+
+    /**
+     * Snaps the element to the target in case of a collision
+     */
+    getSnappedBounds(element: SModelElement & BoundsAware, target: SModelElement & BoundsAware): Bounds {
+        let snappedBounds: Bounds = element.bounds;
```

Comment:
Please avoid specifying the type when it is actually inferred (line `snappedBounds: Bounds`). There are several occurrences of unnecessary type specifications. Please remove them all to make the code more concise.